Functions/Functions/History/UpdateRuleHistory.cs

```diff
@@ -41,19 +41,19 @@ public UpdateRuleHistory(RulesDbContext dbContext)
                     await _dbContext.RuleHistoryCache.Add(new RuleHistoryCache
                     {
                         MarkdownFilePath = historyEntry.file,
-                        ChangedAtDateTime = historyEntry.lastUpdated,
+                        ChangedAtDateTime = DateTime.Parse(historyEntry.lastUpdated),
                         ChangedByDisplayName = historyEntry.lastUpdatedBy,
                         ChangedByEmail = historyEntry.lastUpdatedByEmail,
-                        CreatedAtDateTime = historyEntry.created,
+                        CreatedAtDateTime = DateTime.Parse(historyEntry.created),
```

Comment:
Since you do .ToString(format)

You may as well do .ParseExact(format)

Might be worth abstracting this format to a const string

Reply:
**Done** - Changed `DateTime.Parse` to `DateTime.ParseExact` and moved format to a const string